.goreleaser.yml

```diff
@@ -163,16 +152,14 @@ builds:
     ignore:
       - goos: darwin
         goarch: 386
-      - goos: darwin
-        goarch: arm64
     env:
       - CGO_ENABLED=0
     main: ./cmd/apps/vpn-client/
     ldflags: -s -w -X github.com/skycoin/dmsg/buildinfo.version={{.Version}} -X github.com/skycoin/dmsg/buildinfo.commit={{.ShortCommit}} -X github.com/skycoin/dmsg/buildinfo.date={{.Date}}
 archives:
   - format: tar.gz
     wrap_in_directory: false
-    name_template: 'skywire-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    name_template: 'skywire-{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
```

Comment:
because the generated package name is `skywire-vv4.0.0-Linux-amd64` instead of `skywire-v4.0.0` i tests, not sure if that's what is intended, i'll change it back to it if required.

Reply:
@alexadhy not intended. `skywire-v0.4.0` is intended. I just missed that the `v` will be incldued by `.Version`.